### PR TITLE
Some improvements and updates

### PR DESCRIPTION
**Pull request by mikebaldry**
- relax mongoid constraint
- add name field
- few syntax changes to make it a bit more modern that don't really change anything
- consistent quoting style
- updating generated syntax to be more what we use most of the time
- renamed `one_offs:create_one_off` to `one_offs:create`

### CODE REVIEW

File: lib/one_offs/tasks.rb [hunk 1]
```diff
@@ -2,26 +2,28 @@
...
     ARGV.each { |a| task a.to_sym do ; end }
 
     directory = Rails.root.join('lib', 'one_offs')
     filename = ARGV[1].singularize
 
     File.open("#{directory}/#{Time.now.to_i}_#{filename}.rb", "w") do |file|
-      contents = "class #{filename.classify.to_s}\n"\
-                 "  def self.process\n"\
-                 "    # Enter your migration code here....\n"\
+      contents = "class #{filename.classify}\n"\
```
Comment (markburns) [inline lib/one_offs/tasks.rb:13]: could we use a heredoc?

Comment (markburns) [reply lib/one_offs/tasks.rb:13]: tangential but I don't like the default spec it adds, which has the process in a before block. I'd rather test the change so I always end up deleting that bit. don't suppose you fancy changing that too?

**Review by markburns — APPROVED** [review]

File: lib/one_offs/tasks.rb (hunk 1 above)
Comment (mikebaldry) [reply lib/one_offs/tasks.rb:13]: that drops the new line at the end and doesn't let you indent the inner line of the process method, as well as leaving the start of line indentation there, so you have to outdent the whole thing weirdly. I think it's fine, we rarely if ever have to look at it :D

Comment (markburns) [reply lib/one_offs/tasks.rb:13]: you can do heredoc with squiggles now like 
```
    <<~HEREDOC
      def this_is_still_indented
        correctly
    HEREDOC
```

Comment (mikebaldry) [reply lib/one_offs/tasks.rb:13]: oh, good to know (for the future)

**Pull request merged**